### PR TITLE
Add custom segmentation interface with timestamps

### PR DESCRIPTION
**Pull request by weiglszonja**
Adds EXTRACT segmentation output to `ophys` pipeline.
Uses a custom interface `ScherrerOphysSegmentationInterface` with reusing the imaging timestamps for the segmentation data.

Depends on 
https://github.com/catalystneuro/neuroconv/pull/128 AND https://github.com/catalystneuro/neuroconv/pull/130

### CODE REVIEW

File: src/fee_lab_to_nwb/scherrer_ophys/convert_session.py [hunk 1]
```diff
@@ -3,30 +3,34 @@
...
+ophys_folder_path = Path("/Volumes/t7-ssd/fee-lab-to-nwb/ophys")
+# The timestamp for the recording
+ophys_dataset_timestamp = "2021-07-26T13_50_50"
 
-
-behavior_data_file_path = Path(f"./home_pos-speed-in_{ophys_dataset_timestamp}.csv")
-behavior_movie_file_path = Path(f"./home_arena_{ophys_dataset_timestamp}.avi")
-# Add a description for the behavior video
+# The file path to the behavior movie file
+behavior_movie_file_path = ophys_folder_path / f"./home_arena_{ophys_dataset_timestamp}.avi"
```
Comment (CodyCBakerPhD) [inline src/fee_lab_to_nwb/scherrer_ophys/convert_session.py:18]:
```suggestion
behavior_movie_file_path = ophys_folder_path / f"home_arena_{ophys_dataset_timestamp}.avi"
```

The `/` operator on the `pathlib.Path` object is sufficient for specifying relative locations

File: src/fee_lab_to_nwb/scherrer_ophys/convert_session.py [hunk 2]
```diff
@@ -23,7 +23,9 @@
 
 # The list of file paths to the imaging (.avi) files
 ophys_file_paths = [
-    ophys_file_name for ophys_file_name in ophys_folder_path.iterdir() if ophys_file_name.suffix == ".avi"
+    ophys_file_name
+    for ophys_file_name in ophys_folder_path.iterdir()
+    if ophys_file_name.suffix == ".avi" and ophys_file_name.stem.startswith("invivo")
```
Comment (weiglszonja) [inline src/fee_lab_to_nwb/scherrer_ophys/convert_session.py:28]: you're right @CodyCBakerPhD, in my local setup I filter for files that start with the "invivo" prefix, they should cover all imaging data. how about this?

File: src/fee_lab_to_nwb/scherrer_ophys/convert_session.py (hunk 1 above)
Comment (weiglszonja) [reply src/fee_lab_to_nwb/scherrer_ophys/convert_session.py:18]: Sure, thank you for noticing this!

File: src/fee_lab_to_nwb/scherrer_ophys/convert_session.py (hunk 2 above)
Comment (CodyCBakerPhD) [reply src/fee_lab_to_nwb/scherrer_ophys/convert_session.py:28]: Sure, that works

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**